bleak/backends/bluezdbus/utils.py

```diff
@@ -46,3 +48,10 @@ def format_GATT_object(object_path, interfaces):
     return "\n{0}\n\t{1}\n\t{2}\n\t{3}".format(
         _type, object_path, _uuid, uuidstr_to_str(_uuid)
     )
+
+
+def extract_service_handle_from_path(path):
+    try:
+        return int(_service_handle_regex.search(path).groups()[-1], 16)
+    except Exception as e:
+        raise BleakError(f"Could not parse service handle from path: {path} ({e})")
```

Comment:
Yes. Forgot about this solution. I write few libraries except this and pass on exceptions instead of directly handling them, so I tend to forget to do this...